### PR TITLE
Federated documentation starter

### PR DESCRIPTION
**Pull request by galviset**
This pull request contains all changes needed to start the federated documentation, from the Jekyll configuration to the content of the documentation itself. The shared content is based on EcoPortal's draft documentation which has been slightly edited to be generic enough for most portals.
While most of the shared content is there, the user documentation for each portal needs to be done (either by reusing the available shared content, either by adding their own specific bits of documentation).

### CODE REVIEW

File: docs/users-guide-docs/documentation/earthportal/about.md [hunk 1]
```diff
@@ -5,5 +5,5 @@ permalink: user_guide/EarthPortal
 parent: Users guide
 nav_order: 1
 ---
-{% include documentation_nav.html portal="EarthPortal" %}
+{% include documentation_nav.html %}
```
Comment (Bilelkihal) [inline docs/users-guide-docs/documentation/earthportal/about.md:8]: For the nav, when you do the include you need to explicitly specify the portal (this is why it's not getting colored when you select it),
The config you added do the job only for the paths where we have the shared doc, but for the nav you can't do the same thing

**Comment by Bilelkihal** [comment]
@galviset 
The issue of the last week was not related to using the same page title, it was caused by the includes of the variables not working properly. In fact the config that you did add the values just in the scope of the specific docs, but in our case we need to use the variables in the shared documentation instead and since we have only one shared documentation then we can't really use this solution and we can't run away from passing the variables in each include

**Comment by galviset** [comment]
@Bilelkihal
This is weird. I tried to rollback to the include system of variables (so passing variables in each include and change page. by include. in shared documentation), and I still witness the same issue.
The scope-specific variables were working correctly when I tested them, but it required to change `include.variable` by ` page.variable` in shared docs. Included files still have access to page variables.

**Comment by Bilelkihal** [comment]
No it works perfectly when u add the includes.
you can see also for you that ontoportal and ecoportal has the same page title for home page and it's working.

in this video I just added the includes and now it's working
https://github.com/ontoportal/documentation/assets/61744974/587e1785-1cd2-4dc4-a4e0-66bf0d3f2cf0

**Comment by galviset** [comment]
@Bilelkihal That pull request now contains only the documentation content for review.

**Review by syphax-bouazzouni — CHANGES_REQUESTED** [review]
Hard to review a 175 files change PR, can you divide it 10 small PRs (max 20 files), if you need help on that, we can arrange a meeting.

**Comment by galviset** [comment]
Done, here is the entire content divided:
#55
#56
#57
#58
#59
#60
#61
#62

**Pull request closed without merging**